### PR TITLE
Feature/in process table

### PR DESCRIPTION
**Pull request by nicolascarva**
Naming: feature/descriptive-name

Reviewers: minimum 2 (Chris and/or 2 from Web or DS respectively and not person who submitted PR)

Description: 
  What is the motivation for changes?
  What specific changes were made?

Submitter Checklist: 
  - [x] Small scope (1-2 components)
  - [x] Not too many concerns in a single commit
  - [x] Remove extraneous comments (code in comments), print statements, to-dos, console.logs, extra fluff
  - [x] Atomic, descriptive commit messages
  - [x] Consistent formatting

Reviewer Checklist:
  - [x] Small scope (1-2 components)
  - [x] Comments, print statements, to-dos, console.logs, extra fluff were removed
  - [x] Give feedback no matter what conversations happened elsewhere, document here
  - [x] Comment on the line of code itself (blue plus box) in files changed tab

### CODE REVIEW

File: app/create_db_tables.py [hunk 1]
```diff
@@ -20,12 +20,28 @@ def initialize_police_table():
         title TEXT,
         description TEXT,
         tags TEXT,
-        force_rank TEXT
+        force_rank TEXT,
+        confidence FLOAT
+    )""",
+
```
Comment (jalirkan) [inline app/create_db_tables.py:26]: Is this table title clear?

**Comment by jalirkan** [comment]
Overall, seems good. I wonder if titling a table "in_process" is very clear. Might be worth changing in the future

**Comment by israelaikulola** [comment]
Awesome ! everything looks good

File: app/create_db_tables.py (hunk 1 above)
Comment (nicolascarva) [reply app/create_db_tables.py:26]: We'll leave it as in_process for now, definitely worth changing if we come up with a better name.

**Review by carolinefallonlucas — APPROVED** [review]
Looks good! 😀 Excited to get started on this.

**Pull request merged**